cpp/include/cudf/strings/convert/convert_fixed_point.hpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/strings/strings_column_view.hpp>
+
+namespace cudf {
+namespace strings {
+/**
+ * @addtogroup strings_convert
+ * @{
+ * @file
+ */
+
+/**
+ * @brief Returns a new fixed-point column parsing decimal values from the
+ * provided strings column.
+ *
+ * Any null entries will result in corresponding null entries in the output column.
+ *
+ * Only characters [0-9] plus a prefix '-' and '+' and a single decimal point
+ * are recognized. When any other character is encountered, the parsing ends
+ * for that string and the current digits are converted into a fixed-point value.
```

Comment:
Can you add an example of early termination due to unrecognized character? 

Sounds like "-1.234.45" will result in "-1.234", rather than NULL? Is that the right decision? Either way, I think the docs should have an example of it.

Reply:
I decided to make invalid data format to be undefined behavior rather than try to support this. Hopefully I got the wording right.